messages/create.json

```diff
@@ -1,7 +1,7 @@
 {
   "example": "$ sfdx force:lightning:lwc:test:create -f force-app/main/default/lwc/myButton/myButton.js",
-  "commandDescription": "create a Lightning web component test with boilerplate code inside a __tests__ directory",
-  "longDescription": "Creates a __tests__ directory in the specified directory. Creates a boilerplate yourComponentName.test.js file in the __tests__ directory.",
+  "commandDescription": "creates a Lightning web component test file with boilerplate code inside a __tests__ directory.",
+  "commandLongDescription": "Creates a __tests__ directory in the specified directory. Creates a yourComponentName.test.js file with boilerplate code in the __tests__ directory.",
```

Comment:
Do you know if this is automatically consumed somewhere by SFDX? Some of the messages are directly imported by the plugin, but I didn't see anything directly importing `longDescription`.

Reply:
I get the impression from this doc
https://developer.salesforce.com/docs/atlas.en-us.sfdx_cli_plugins.meta/sfdx_cli_plugins/cli_plugins_style_help.htm
that _commandLongDescription_ is automatically consumed and ends up in the output here https://developer.salesforce.com/docs/atlas.en-us.sfdx_cli_reference.meta/sfdx_cli_reference/cli_reference_force_lightning.htm#cli_reference_force_lightning
and it seems (again, hard to tell for sure) like if you don't have a _commandLongDescription_ it uses the _commandDescription_ (short one). I don't see _longDescription_ showing up anywhere.